CVChatbot/CVChatbot.Bot/ChatbotActions/Commands/Stats.cs

```diff
@@ -17,9 +17,9 @@ public override void RunAction(ChatExchangeDotNet.Message userMessage, ChatExcha
 
             var statsTable = doc.DocumentNode
                 .Descendants("table")
-                .Where(x => x.Attributes["class"] != null)
-                .Where(x => x.Attributes["class"].Value == "task-stat-table")
-                .Single();
+                .Single(x => 
+                    x.Attributes["class"].Value == "task-stat-table" && 
+                    x.Attributes["class"] != null);
```

Comment:
Well i'm fine with this for now. Eventually I would like to know the answer (so I can change my practices).

The only thing I've dug up so far is [a Jon Skeet answer](http://stackoverflow.com/questions/4735531/is-multiple-where-statements-in-linq-a-performance-issue) but even that is a bit wishy washy.